### DISCUSSION TITLE
Welcome to ohmyform Discussions!

### DISCUSSION THREAD

**Discussion by wodka** [ohmyform/ohmyform#132] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by githubber** [comment]
Hi! Great app, thank you, and thank you too to CapRover for making it easy to host.
I've tried to embed a form in a Wordpress site but it doesn't work, failure unknown.
Does anyone know how to do this please?
Thank you!

**Comment by ctorres-dev** [comment]
hello,

I have installed ohmyform in a dedicated server and I try to activate the api to be able to create forms from external or by terminal and it does not allow it, besides not being able to edit the permissions of the users, to somebody the same thing happened to him.
